ng/src/web/components/table/head.js

```diff
@@ -38,20 +40,48 @@ const TableHead = ({
     className,
     colSpan,
     rowSpan,
-    sortby,
+    sortDir,
+    filterSortBy,
```

Comment:
Could you please rename it to sortOrder. For me it is not consistent to name it filterSortBy and also have sortDir which comes from the filter too.